### DISCUSSION TITLE
Calculating Learning rate of a NN for a time series data using LSTM

### DISCUSSION THREAD

**Discussion by cnrgbot** [nengo/nengo-zoo#1060] [FAQ]
> _Archived from the Nengo Discourse forum._ Originally posted by **MNHaque** on 2022-01-03. (Forum topic #2013, post #1 / id 6168)

Hi Everyone,

Can anyone please help me with a source/reference code to calculate the learning rate of a neural network for a time series data using LSTM? It would be highly appreciated. Thank you.

**Comment by cnrgbot** [comment]
> _Archived from the Nengo Discourse forum._ Originally posted by **xchoo** on 2022-01-07. (Forum topic #2013, post #2 / id 6186)

Hi @​MNHaque,

Your question is a little out of the scope of these forums, but I'll try my best to answer it. From my consultation with the other ML experts, they suggested to use a learning rate scheduler with your LSTM model. I did a quick google and found a few tutorials ([here](https://towardsdatascience.com/learning-rate-schedules-and-adaptive-learning-rate-methods-for-deep-learning-2c8f433990d1) and [here](https://machinelearningmastery.com/using-learning-rate-schedules-deep-learning-models-python-keras/)) explaining this concept. :slight_smile:
